lerna.json

```diff
@@ -14,30 +14,9 @@
   "lerna": "3.2.1",
   "npmClient": "npm",
   "packages": [
-    "libraries/botbuilder",
-    "libraries/botbuilder-ai",
-    "libraries/botbuilder-ai-orchestrator",
-    "libraries/botbuilder-applicationinsights",
-    "libraries/botbuilder-azure",
-    "libraries/botbuilder-core",
-    "libraries/botbuilder-dialogs",
-    "libraries/botbuilder-dialogs-adaptive",
-    "libraries/botbuilder-dialogs-adaptive-testing",
-    "libraries/botbuilder-dialogs-declarative",
-    "libraries/botbuilder-testing",
-    "libraries/botbuilder-lg",
-    "libraries/adaptive-expressions",
-    "libraries/botframework-config",
-    "libraries/botframework-connector",
-    "libraries/botframework-schema",
-    "libraries/functional-tests",
-    "libraries/functional-tests/dialogToDialog/dialogRootBot",
-    "libraries/functional-tests/dialogToDialog/dialogSkillBot",
-    "libraries/testbot",
-    "libraries/botframework-streaming",
-    "libraries/bot-integration-tests",
-    "libraries/testskills/skillparent",
-    "libraries/testskills/skillchild",
+    "libraries/*",
+    "libraries/functional-tests/dialogToDialog/*",
+    "libraries/testskills/*",
```

Comment:
One thing I'd like to do in the future is move or consolidate the test projects. It bothers me that they're in the `libraries/` folder. Low pri issue.